src/main/resources/application.yaml

```
@@ -202,3 +202,6 @@ clients:
   azure-graph:
     uri: https://graph.microsoft.com/v1.0/
     scope: https://graph.microsoft.com/.default
+  dp:
+    uri: https://www.nav.no/dagpenger/datadeling
+    scope: api://${CLIENT_ENV}-fss.teamdagpenger.dp-proxy/.default
```

Scope her må være for dp-datadeling, scopet brukes for å utstede en token.
Så her ønsker man fra integrasjoner å utstede token for applikasjon X
```suggestion
    scope: api://${CLIENT_ENV}-gcp.teamdagpenger.dp-datadeling/.default
```